### PR TITLE
refactor(example): use barrel import for book module

### PR DESCRIPTION
**Pull request by jasonhodges**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
relates to #1808 

Closes #

## What is the new behavior?
This PR introduces barrel imports for book module and its sub-folders.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/example-app/src/app/books/actions/book.actions.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
+import { Book } from '@example-app/books/models';
```
Comment (brandonroberts) [inline projects/example-app/src/app/books/actions/book.actions.ts]: Don't re-order the imports. I believe the style guide says order them by name, but they are fine as-is. Its preference, but we order them this way with a line break in between each set.

1. Framework imports (@angular/core)
2. Third-party imports (@angular/material, @ngrx/store)
3. Local imports (@example-app)

Comment (jasonhodges) [reply projects/example-app/src/app/books/actions/book.actions.ts]: Ok, no problem @brandonroberts 
That was WebStorm's 'optimize imports' formatting

Comment (jasonhodges) [reply projects/example-app/src/app/books/actions/book.actions.ts]: @brandonroberts I reordered the imports per preference you mentioned previously. Let me know if we are clear to merge. Thanks

**Comment by brandonroberts** [comment]
@jasonhodges the `yarn.lock` file needs to be removed. Everything else LGTM

**Comment by jasonhodges** [comment]
> @jasonhodges the new `yarn.lock` file needs to be removed. Everything else LGTM

Sorry about that! Deleted

**Comment by brandonroberts** [comment]
Sorry about that. Revert the changes to the `yarn.lock` so that it doesn't appear in the diff. I though there was a new one

**Pull request merged**